.github/workflows/thirdparty.yaml

```diff
@@ -0,0 +1,39 @@
+name: Third Party Tests
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - "*"
+
+jobs:
+  third-party-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: ['1.22.x']
+
+    steps:
+      - name: Checkout parquet-go
+        uses: actions/checkout@v3
+
+      - name: Clone Tempo
+        run: git clone https://github.com/grafana/tempo.git
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: ${{ matrix.go }}
+
+      - name: Replace parquet-go with local version
+        run: |
+          cd tempo
+          go mod edit -replace github.com/parquet-go/parquet-go=../parquet-go
+
+      - name: Run Critical Tempo Tests
+        run: |
+          cd tempo
+          go test ./tempodb/encoding/vparquet4/...
```

Comment:
This is amazing. Thank you! 

I have one ask if its not too difficult. Can we make the script test all folders that match `./tempodb/encoding/vparquet*`?